R/expect-self-test.R

```diff
@@ -67,7 +67,8 @@ show_failure <- function(expr) {
 #' @rdname expect_success
 #' @param path Path to save failure output
 expect_known_failure <- function(path, expr) {
-  FailureReporter <- R6::R6Class("FailureReporter", inherit = CheckReporter,
+  FailureReporter <- R6::R6Class(
+    "FailureReporter", inherit = CheckReporter,
```

Comment:
Why doesn't `"FailureReporter"` stay on the first line? (Given that it's an unnamed argument that fits)

Reply:
Because that is not yet implemented in styler. See https://github.com/krlmlr/styler/issues/246. For now, the line is broken after the first parenthesis with `strict = TRUE` if the whole call does not fit on one line.